.github/workflows/bump-version-release.yaml

```diff
@@ -53,11 +53,12 @@ jobs:
       old-version: ${{ steps.bump-version.outputs.old-version }}
 
     steps:
-      - name: Check out repository
-        uses: bakdata/ci-templates/actions/checkout@1.32.0
-        with:
-          ref: ${{ github.event.repository.default_branch }}
-          persist-credentials: false # required for pushing to protected branch later
+      # assume that the repository is already checked out
+      # - name: Check out repository
+      #   uses: bakdata/ci-templates/actions/checkout@1.32.0
+      #   with:
+      #     ref: ${{ github.event.repository.default_branch }}
+      #     persist-credentials: false # required for pushing to protected branch later
```

Comment:
I think the main reason this is in here is this persist-credentials flag. Does it hurt to keep it hear even if we are checking out the repo again?

Reply:
I dont think it hurts, ill add it back in